support/ab.c

```diff
@@ -1870,30 +1871,29 @@ static void test(void)
     }
 
     /* setup request */
+    const char *const http_version = http_spec == HTTP_1_1 ? "HTTP/1.1" : "HTTP/1.0";
+    const char *const request_uri = isproxy ? fullurl : path;
+    const char *const connection = keepalive ? "Connection: Keep-Alive\r\n" 
+                                             : http_spec == HTTP_1_1 ? "Connection: Close\r\n" : "";
```

Comment:
This possibly could be:
```
    const char *const connection =
        http_spec == HTTP_1_0 && keepalive ? "Connection: Keep-Alive\r\n"
      : http_spec == HTTP_1_1 && !keepalive ? "Connection: Close\r\n"
      : "";
```
The rationale being that Keep-Alive is implicitely disabled for HTTP/1.0 and enabled for HTTP/1.1.